README.md

```diff
@@ -139,13 +139,8 @@ There is `Vagrantfile` provided with braid, that will setup a staging sever.
 It uses the address `172.16.255.140`, and there is a braid config named `vagrant` that connects to it by default.
 
 ```shell
-# Get the required base OS image if you don't have it yet.
-vagrant box add precise64 http://files.vagrantup.com/precise64.box
```

Comment:
Maybe we should document that vagrant 1.7.3 or never is required... or instruct users how to manually get the image if they have older versions.